src/test/java/com/google/api/codegen/testsrc/library_pkg2.yaml

```diff
@@ -0,0 +1,11 @@
+api_name: library
+api_version: v1
+artifact_type: GAPIC
+organization_name: google-cloud
+proto_path: google/library
+proto_deps:
+- google-common-protos
+- google-some-other-package-v1
+proto_test_deps:
+- google-some-test-package-v1
+release_level: GA
```

Comment:
Would changing the release level here fix the baseline changes in dynamic langs? Or would it just cause other baseline changes elsewhere?

Reply:
Yeah it would make other baselines change (e.g. Java) - we previously had different inputs for different languages, and now this makes them all the same.